### PR TITLE
Update visual cases

### PR DESCRIPTION
**Pull request by yutannihilation**
It seems these are diffs about coloubar legends, though they doesn't seem different to my eyes.

The latest build passes (https://github.com/tidyverse/ggplot2/commit/cb0eaa0e16063517e7c8f8453bb2a10641d2f0d1) on 6 days ago, so I suspect this might be related to some update of scales.

### CODE REVIEW

File: tests/figs/guides/white-to-red-gradient-colorbar-white-tick-marks-no-frame.svg [hunk 1]
```diff
@@ -51,7 +51,7 @@
...
 <g clip-path='url(#cpMC4wMHw3MjAuMDB8NTc2LjAwfDAuMDA=)'><text transform='translate(13.04,286.70) rotate(-90)' style='font-size: 11.00px; font-family: Liberation Sans;' textLength='5.50px' lengthAdjust='spacingAndGlyphs'>x</text></g>
 <rect x='679.54' y='233.09' width='34.98' height='101.72' style='stroke-width: 1.07; stroke: none; fill: #FFFFFF;' clip-path='url(#cpMC4wMHw3MjAuMDB8NTc2LjAwfDAuMDA=)' />
-<g clip-path='url(#cpMC4wMHw3MjAuMDB8NTc2LjAwfDAuMDA=)'><image width='17.28' height='86.40' x='679.54' y='248.41' xlink:href='data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABIAAABXCAYAAAAJWxWIAAAABmJLR0QA/wD/AP+gvaeTAAABE0lEQVRYhc2YwRHDMAgEF4Q9frmvVJKGUw/k4RK8D7kAzbHHCeSYM4a3X8ynuOv1OcRQXOf7g4DiEhTNUJxWaceSFB2WohIUhaVohmIJihiKSuGcoCijtKBYhiLLNQ32YLmGxCiSYhkRsVzL7WB7rlmMsBjFUOROjAaKNDq7d4tIW6VlWPanpOix35priv0ao7AYjcQIjZEV2gzpGsFyjZYYdUqlaYy0zm4r/YlVmtWQbcHe9KpVsmYdpDEa7RppqzTLNawBibVEPOkXFKU1IMPaalvrbG0ZtWCPC9uw39pGQGIUlmvjbiPaFNGeWTuN7LAuf22KeLC1XxrWcz2srGnvNSzX2gptWqHF6qN+7P8Kgn5/vNJD8shvDpsAAAAASUVORK5CYII='/></g>
```
Comment (yutannihilation) [inline tests/figs/guides/white-to-red-gradient-colorbar-white-tick-marks-no-frame.svg]: left: before, right: after

Screenshot from 2019-11-24 17-29-47

File: tests/figs/guides/white-to-red-gradient-colorbar-thick-black-tick-marks-green-frame.svg [hunk 2]
```diff
@@ -51,7 +51,7 @@
...
 <g clip-path='url(#cpMC4wMHw3MjAuMDB8NTc2LjAwfDAuMDA=)'><text transform='translate(13.04,286.70) rotate(-90)' style='font-size: 11.00px; font-family: Liberation Sans;' textLength='5.50px' lengthAdjust='spacingAndGlyphs'>x</text></g>
 <rect x='679.54' y='233.09' width='34.98' height='101.72' style='stroke-width: 1.07; stroke: none; fill: #FFFFFF;' clip-path='url(#cpMC4wMHw3MjAuMDB8NTc2LjAwfDAuMDA=)' />
-<g clip-path='url(#cpMC4wMHw3MjAuMDB8NTc2LjAwfDAuMDA=)'><image width='17.28' height='86.40' x='679.54' y='248.41' xlink:href='data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABIAAABXCAYAAAAJWxWIAAAABmJLR0QA/wD/AP+gvaeTAAABE0lEQVRYhc2YwRHDMAgEF4Q9frmvVJKGUw/k4RK8D7kAzbHHCeSYM4a3X8ynuOv1OcRQXOf7g4DiEhTNUJxWaceSFB2WohIUhaVohmIJihiKSuGcoCijtKBYhiLLNQ32YLmGxCiSYhkRsVzL7WB7rlmMsBjFUOROjAaKNDq7d4tIW6VlWPanpOix35priv0ao7AYjcQIjZEV2gzpGsFyjZYYdUqlaYy0zm4r/YlVmtWQbcHe9KpVsmYdpDEa7RppqzTLNawBibVEPOkXFKU1IMPaalvrbG0ZtWCPC9uw39pGQGIUlmvjbiPaFNGeWTuN7LAuf22KeLC1XxrWcz2srGnvNSzX2gptWqHF6qN+7P8Kgn5/vNJD8shvDpsAAAAASUVORK5CYII='/></g>
```
Comment (yutannihilation) [inline tests/figs/guides/white-to-red-gradient-colorbar-thick-black-tick-marks-green-frame.svg]: left: before, right: after

Screenshot from 2019-11-24 17-29-28

File: tests/figs/guides/one-combined-colorbar-for-colour-and-fill-aesthetics.svg [hunk 3]
```diff
@@ -51,7 +51,7 @@
...
 <g clip-path='url(#cpMC4wMHw3MjAuMDB8NTc2LjAwfDAuMDA=)'><text transform='translate(13.04,286.70) rotate(-90)' style='font-size: 11.00px; font-family: Liberation Sans;' textLength='5.50px' lengthAdjust='spacingAndGlyphs'>y</text></g>
 <rect x='686.87' y='233.09' width='27.65' height='101.72' style='stroke-width: 1.07; stroke: none; fill: #FFFFFF;' clip-path='url(#cpMC4wMHw3MjAuMDB8NTc2LjAwfDAuMDA=)' />
-<g clip-path='url(#cpMC4wMHw3MjAuMDB8NTc2LjAwfDAuMDA=)'><image width='17.28' height='86.40' x='686.87' y='248.41' xlink:href='data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABIAAABXCAYAAAAJWxWIAAAABmJLR0QA/wD/AP+gvaeTAAABjUlEQVRYhc2Yy3EdMRADG9SUHZsjcVzKyZH4YFtLcqDDcwaDgzaArh6Any3qnHcz/Oz+Udr/phwsEwGVobQ/xiCA4pkbYVEkjCSKQEa4QyCcygiKE8jIhEaTQqPRFPvvnCNCIJtyACSJ8vNnDLpAsX+PQQiqEyBDeX9PCFF9vs2NJMq75hxDdQBkoPq8zY1erc1BWJQDILGo3msutKD6zEFqZUDYVD9z0JIoB4xOm+pHY5AUau2V0Z4bLd4yoF5kWjOmzk2AoPYNnUe3A61JISOJOgGjtqlz5+eRbep0ACSoHTASZDKSO5PRWitj5A7VbzsTNsuhsFvUjuz+kJFXakG2qOPALXKhdo85rxPyBEA4ZCSFjNbLaB62ZeqM3yGA/moZdao1BLUDGanJhL1IjbZSo53Upk2FTUM9KaPDnZNY1FnPnGNRlwAIqKMAaHXKSNQNGBkyRq+blv11jBpTxwEjieqA0f+f0bmRENV9xiCWqJvI6Jg6dw7Cop4OLEhBfXTgzd+itvfPMYn96xMzVA/gKz5U8AAAAABJRU5ErkJggg=='/></g>
```
Comment (yutannihilation) [inline tests/figs/guides/one-combined-colorbar-for-colour-and-fill-aesthetics.svg]: left:before, right: after

Screenshot from 2019-11-24 17-29-10

**Comment by hadley** [comment]
You can click this button — Screenshot 2019-11-25 at 7 10 48 AM — it often doesn't work the first time for me, but if you toggle back to source diff and then again to rich diff it does work.

**Comment by hadley** [comment]
This is probably due to some subtly different calculation by farver

**Comment by yutannihilation** [comment]
> but if you toggle back to source diff and then again to rich diff it does work.

Oh, thanks for this tips! (But, this time this doesn't work for me...) I'll try next time :)

**Pull request merged**